sqldata-mycollection.sql

```diff
@@ -0,0 +1,65 @@
+# ************************************************************
+# Sequel Ace SQL dump
+# Version 20046
+#
+# https://sequel-ace.com/
+# https://github.com/Sequel-Ace/Sequel-Ace
+#
+# Host: 127.0.0.1 (MySQL 5.5.5-10.11.2-MariaDB-1:10.11.2+maria~ubu2204)
+# Database: mycollection
+# Generation Time: 2023-04-03 09:47:45 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+SET NAMES utf8mb4;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE='NO_AUTO_VALUE_ON_ZERO', SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table collection_recipes
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `collection_recipes`;
+
+CREATE TABLE `collection_recipes` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `date` date NOT NULL,
+  `description` varchar(255) NOT NULL,
+  `imagelink` varchar(255) DEFAULT NULL,
+  `ingreds` varchar(255) DEFAULT NULL,
+  `recipe` varchar(512) DEFAULT NULL,
```

Comment:
You might want to add more space for description, recipe and incredients. I could see them filling up very quickly.